scipy/optimize/_linprog_highs.py

```diff
@@ -200,14 +208,17 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
     _check_unknown_options(unknown_options)
 
     # Map options to HiGHS enum values
-    simplex_dual_edge_weight_strategy = _convert_to_highs_enum(
+    if simplex_dual_edge_weight_strategy is None:
+        simplex_dual_edge_weight_strategy = 'steepest-devex'
+    simplex_dual_edge_weight_strategy_enum = _convert_to_highs_enum(
         simplex_dual_edge_weight_strategy,
         'simplex_dual_edge_weight_strategy',
         choices={'dantzig': HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_DANTZIG,
                  'devex': HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_DEVEX,
                  'steepest-devex': HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STEEP2DVX,
                  'steepest':
-                 HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_STEEPEST_EDGE})
+                 HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_STEEPEST_EDGE,
+                 None: HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STEEP2DVX})
```

Comment:
```suggestion
                 None: HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_CHOOSE})
```
But I couldn't import `HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_CHOOSE`

Reply:
As of tonight, HIGHS_SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_CHOOSE exists only as an idea - a means to simplify the edge weight strategy names. I'll let you know when it's in master

Note that Gurobi doesn't distinguish primal and dual in its weight strategies. See

https://www.gurobi.com/documentation/9.0/refman/simplexpricing.html

where "Partial Pricing" is a primal-only technique, and "Quick-Start Steepest Edge" is (roughly) my SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_STEEPEST_EDGE_UNIT_INITIAL

Reply:
@mdhaber as @jajhall mentioned, `SIMPLEX_DUAL_EDGE_WEIGHT_STRATEGY_CHOOSE` is not currently available in master, so probably the best thing to do here is map `None` to `None` and the wrapper will select the default HiGHS value

Reply:
I see you did that. Thanks.